plugin/pkg/admission/resourcequota/apis/resourcequota/v1beta1/BUILD

```diff
@@ -16,7 +16,7 @@ go_library(
         "zz_generated.deepcopy.go",
         "zz_generated.defaults.go",
     ],
-    importpath = "k8s.io/kubernetes/plugin/pkg/admission/resourcequota/apis/resourcequota/v1alpha1",
```

Comment:
Please do not delete (rename) v1alpha1 files. If we deleted these files, v1alpha1 APIs would stop working in the next K8s release.